src/test/java/io/stargate/sgv2/jsonapi/exception/JsonApiExceptionTest.java

```diff
@@ -13,7 +13,7 @@
 class JsonApiExceptionTest {
   @Test
   public void happyPath() {
-    JsonApiException ex = new JsonApiException(ErrorCodeV1.COMMAND_UNKNOWN);
```

Comment:
Silly test, but had to change to use `ErrorCodeV1` entry that was not being converted.